plugin/src/sbt-test/shocon/basic/build.sbt

```diff
@@ -1,29 +1,38 @@
+// https://github.com/portable-scala/sbt-crossproject
+// (5) shadow sbt-scalajs' crossProject and CrossType until Scala.js 1.0.0 is released
+import sbtcrossproject.{crossProject, CrossType}
 
-lazy val root = crossProject.in(file("."))
+lazy val root = crossProject(JSPlatform, JVMPlatform, NativePlatform).in(file("."))
   .dependsOn(lib)
   .enablePlugins(ShoconPlugin)
   .settings(
-    scalaVersion := "2.12.2",
+    scalaVersion := "2.11.12",
     name := "basic",
     version := "0.1.0-SNAPSHOT",
     description := "Basic test for the shocon sbt plugin",
-    libraryDependencies += "eu.unicredit" %%% "shocon" % sys.props.getOrElse("plugin.version", sys.error("'plugin.version' environment variable is not set")),
+    libraryDependencies += "org.akka-js" %%% "shocon" % sys.props.getOrElse("plugin.version", sys.error("'plugin.version' environment variable is not set")),
```

Comment:
good catch! thanks! wondering why it was working before ... uhmmm ..

Reply:
`plugin/scripted` was not part travis build

Reply:
correct